legacyman_parser/parse_flag_of_country.py

```diff
@@ -115,7 +115,8 @@ def extract_flag_of_ns_country(soup: BeautifulSoup = None, parsed_url: str = Non
     # Copy to target directory
     destination_file = COPY_FLAGS_TO_DIRECTORY + \
         userland_dict['country'].country.lower() + os.path.splitext(flag_image)[1].lower()
-    shutil.copy2(flag_image, destination_file)
+    # shutil.copy2(flag_image, destination_file)
```

Comment:
hello @brahnavank - is there a reason why we're using the operating system copy instead of the Python one?

Hmm, actually... since the target operating system is MS-Windows we'll need to use the Python version.

I think that the `dita` validate call will still work though, since I'm pretty sure it will just pick up the `dita.bat` version.